src/store/reducers/dform/initial/styles.json

```diff
@@ -1735,6 +1735,7 @@
                   "field": "pathology.haematology.hemoglobin",
                   "label": "Hemoglobin (Hb)",
                   "type": "number", "min": 0, "suffix": "g/dl",
+                  "normalRange": [ 12, 18 ],
```

Comment:
I see! I just checked this and it seems there is function `findActiveAlert()` in `fields/TextInput/index.js` which can take alerts prop or normalRange prop. 
I will remove alerts from all fields with a normalRange to keep with DRY.

Reply:
(I keep both fields only for haemotology.wbc field, as normalRange has formatted strings to better show on print form, and alerts contains the numeric values. What do you think?)

Reply:
I respect your decision!

Reply:
I think, in that case, `normalRange` should be defined as numbers and formatted at rendering because `normalRange` prop is for numerical validation rather than rendering.